openapi/auth-server.yaml

```diff
@@ -375,6 +376,27 @@ components:
         - type
         - actions
         - identifier
+    access-identity:
```

Comment:
Based on the discussion from the Open Payments Community call, this would also require an additional property that will need to be provided - the user wallet address. This way, the wallet provider can validate if the connected user (on their side) is the owner of the wallet address when the user is redirected to the IdP page.